common/nymsphinx/params/src/packet_modes.rs

```diff
@@ -11,7 +12,7 @@ pub struct InvalidPacketMode {
 }
 
 #[repr(u8)]
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Default)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Default, Serialize, Deserialize)]
 pub enum PacketMode {
```

Comment:
What you could do is rename it to `PacketType`, rename `Mix` to `Sphinx` and remove `Vpn` (since nobody is or will be using it in this form). However, keep the `Outfox` value at `2`.. Then this field will no longer be abused yet be still fully backwards compatibile